<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>主資料庫假日設定</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QDateEdit" name="qtDateEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>理由</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="qtReasonLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="qtHolidayRadioButton">
       <property name="text">
        <string>放假</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="qtWorkdayRadioButton">
       <property name="text">
        <string>補班</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="qtAddPushButton">
       <property name="text">
        <string>新增</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="qtExportPushButton">
       <property name="text">
        <string>匯出</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="qtImportPushButton">
       <property name="text">
        <string>匯入</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="qtExitPushButton">
       <property name="text">
        <string>離開</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
